<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UpdEmp</class>
 <widget class="QDialog" name="UpdEmp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>160</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Employee Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>User Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Password:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Employee Type:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Mail Address:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Hour Rate:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Salary:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Hour Limit:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Commission Rate:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Deduction</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="buttonUpdate">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>update</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>40</y>
     <width>160</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="lineEditEmployeeName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditUserName"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditPassword"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditEmployeeType"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditMailAddress"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditHourRate"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditSalary"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditHourLimit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCommRate"/>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDeduction"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
